packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -268,8 +272,25 @@ const App = (props) => {
         const path = buildUrl('/account/wishlist')
         history.push(path)
     }
+
+    const trackPage = () => {
+        activeData.trackPage(site.id, locale.id, currency)
+    }
+
+    useEffect(() => {
+        trackPage()
+    }, [location])
+
     return (
         <Box className="sf-app" {...styles.container}>
+            <Helmet>
+                {ACTIVE_DATA_ENABLE == 1 && (
```

Comment:
After converting to `ACTIVE_DATA_ENABLE = true` we can avoid the suboptimal usage of `==` which is not a good comparator in javascript
```suggestion
                {ACTIVE_DATA_ENABLE && (
```